plugins/com.google.cloud.tools.eclipse.ui.util/src/com/google/cloud/tools/eclipse/ui/util/WorkbenchUtil.java

```diff
@@ -43,4 +57,56 @@ public static void openInEditor(IWorkbench workbench, IFile file) {
     }
   }
   
+  /**
+   * Opens the specified url in a Web browser instance in a UI thread.
+   *
+   * @param urlPath the URL to display
+   * @param browserId if an instance of a browser with the same id is already opened, it will be
+   *   returned instead of creating a new one. Passing null will create a new instance with a
+   *   generated id.
+   * @param name a name used for the presentation of the internal browser
```

Comment:
I'm sorry, but I still don't understand what the phrase "presentation of the internal browser" is trying to say. Please clarify in the doc comment.